packages/react-dom/src/events/ReactDOMEventReplaying.js

```diff
@@ -477,6 +475,7 @@ function attemptReplayContinuousQueuedEvent(
       queuedEvent.blockedOn = nextBlockedOn;
       return false;
     }
+    replayEvent(queuedEvent, targetContainer);
```

Comment:
Doesn't this need `nextBlockedOn` (or, specifically, whether it's `=== undefined`) to know whether to dispatch with or without the target?

Reply:
Would be good to have some test coverage at least for one of those cases. So we can check it's all being passed through.